source/views/DateView.mc

```diff
@@ -13,7 +14,8 @@ class DateView extends Components.Box {
         var posY = self.getPosY();
         var width = self.getWidth();
 
-        var dateObj = formatDateToEng(Time.now());
+        var getDateFormat = SettingsModule.getValue(SettingsModule.SettingType.DATE_FORMAT);
```

Comment:
Could you please update the way you get and save setting as in this commit https://github.com/shbumba/SimplePixels/commit/10695146e30991df85a176e0e6f48b35a05b3258#diff-56676d6adbb568c19d7318b283ca7174b643c7884ef7c1520dbc5da0dd534b26R30

The thing is that getting settings from Properties every time causes parsing of values from file storage, which causes excessive load on older devices

Reply:
OK.